### ISSUE TITLE
Can the network structure recognize Chinese?

### ISSUE THREAD

**Issue by litetoooooom** [Jyouhou/ICDAR2019-ArT-Recognition-Alchemy#12]

**Comment by longshangbang** [comment]
Yes sure. You can change the vocabulary list and find suitable training data :)

**Comment by litetoooooom** [comment]
> Yes sure. You can change the vocabulary list and find suitable training data :)

thanks for your reply.  
I added the "CHINESE" dataset and the corresponding code section. Use the following command to train and test, and can't get the recognized text output, only the picture in "chinese_rectified".

train cmd:
python examples/main.py --logs_dir logs/test --CRNN --train_data_dir ./datasets/CHINESE --test_data_dir ./datasets/CHINESE_test --test_dataset chinese --voc_type CHINESE --epochs 10
test cmd: 
python examples/main.py --evaluate --tps_margins 0.01 0.01 --arch ResNet_FPN --CRNN --logs_dir logs/test --resume logs/test/model_best.pth.tar --test_data_dir ./datasets/CHINESE_test --test_dataset chinese --voc_type CHINESE  --vis_dir chinese_rectified

**Comment by longshangbang** [comment]
One possible reason lies in the post-processing part. 

Note that the English benchmark datasets are not labelled with punctuations, so I "normalize" the output by removing non-alphanumeric characters. I am not sure if you have modified this part as well?  Check here: https://github.com/Jyouhou/ICDAR2019-ArT-Recognition-Alchemy/blob/911c572c2aff4599a74b7974d46ef4cfb17078b9/Source/evaluation_metrics/metrics.py#L21

**Comment by longshangbang** [comment]
BTW, is the loss decreasing at a reasonable pace?

**Comment by litetoooooom** [comment]
After modifying the normalization function. the loss of the model seems to be normal, the test accuracy is 95%, but it is very bad in the actual data.

**Comment by xxxpsyduck** [comment]
@chinazoujy How bad is it? Did you train on synthetic data?

**Comment by xxxpsyduck** [comment]
@Jyouhou I want to implement this to other non-latin language. Which parts should I modify? Should I train a separate one for numeric characters?

**Comment by longshangbang** [comment]
> After modifying the normalization function. the loss of the model seems to be normal, the test accuracy is 95%, but it is very bad in the actual data.

What test data are you using? It will be more helpful if you can share some examples.

**Comment by longshangbang** [comment]
mainly the vocabulary part and data loader part.
